event_routing_backends/models.py

```diff
@@ -240,6 +242,16 @@ def _match_event_for_host(self, original_event, host_config):
             bool
         """
         for key, value in host_config.get('match_params', {}).items():
-            if get_value_from_dotted_path(original_event, key) != value:
+            original_event_value = get_value_from_dotted_path(original_event, key)
+            if isinstance(value, list):
+                matched = False
+                for value_item in value:
+                    if bool(re.compile(value_item)) and re.search(value_item, original_event_value):
```

Comment:
It would be nice to log the instance if invalid regex is encountered.

Reply:
@ayub02 In that case we are going for exact match. So I don't think we need to log

Reply:
The idea is to have some way for administrator to find out that they have used invalid regex in configuration.

Reply:
@ayub02 its done